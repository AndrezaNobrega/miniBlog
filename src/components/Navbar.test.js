import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from './Navbar';

test('matches snapshot', () => {
  const { asFragment } = render(
    <Router> 
      <Navbar />
    </Router>
  );
  expect(asFragment()).toMatchSnapshot();
});
