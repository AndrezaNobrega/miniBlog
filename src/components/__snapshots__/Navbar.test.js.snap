// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="navbar"
  >
    <a
      aria-current="page"
      class="brand active"
      href="/"
    >
      Mini 
      <span>
        Blog
      </span>
    </a>
    <ul
      class="links_list"
    >
      <li>
        <a
          aria-current="page"
          class="active"
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          class=""
          href="/login"
        >
          Entrar
        </a>
      </li>
      <li>
        <a
          class=""
          href="/register"
        >
          Registrar
        </a>
      </li>
      <li>
        <a
          class=""
          href="/about"
        >
          Sobre
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
